sardana_albaem/ctrl/Albaem2OneDCtrl.py

```diff
@@ -0,0 +1,486 @@
+
+                # NOTE: EM MAY ANSWER WITH MULTIPLE ANSWERS IN CASE OF AN
+                # EXCEPTION
+                # SIMPLY GET THE LAST ONE
+                if data.count(';') > 1:
+                    data = data.rsplit(';')[-2:]
+                return data[:-2]
+
+###############################################################################
+#                Axis Extra Attribute Methods
+###############################################################################
+
+    @debug_it
+    @handle_error(msg="get_Range:")
+    def get_Range(self, axis):
+        if axis == 1:
+            raise RuntimeError('The axis 1 does not use the extra attributes')
+        axis -= 1
+        cmd = 'CHAN{0:02d}:CABO:RANGE?'.format(axis)
+        return self.sendCmd(cmd)
+
+    @debug_it
+    @handle_error(msg="set_Range:")
+    def set_Range(self, axis, value):
+        if axis == 1:
+            raise RuntimeError('The axis 1 does not use the extra attributes')
+        axis -= 1
+        cmd = 'CHAN{0:02d}:CABO:RANGE {1}'.format(axis, value)
+        self.sendCmd(cmd)
+
+    @debug_it
+    @handle_error(msg="get_Inversion:")
+    def get_Inversion(self, axis):
+        if axis == 1:
+            raise RuntimeError('The axis 1 does not use the extra attributes')
+        axis -= 1
+        cmd = 'CHAN{0:02d}:CABO:INVE?'.format(axis)
+        val = self.sendCmd(cmd)
+        if val.lower() == 'off':
+            ret = False
+        elif val.lower() == 'on':
+            ret = True
+        return ret
+
+    @debug_it
+    @handle_error(msg="set_Inversion:")
+    def set_Inversion(self, axis, value):
+        if axis == 1:
+            raise RuntimeError('The axis 1 does not use the extra attributes')
+        axis -= 1
+        cmd = 'CHAN{0:02d}:CABO:INVE {1}'.format(axis, int(value))
+        self.sendCmd(cmd)
+
+    @debug_it
+    @handle_error(msg="get_InstantCurrent:")
+    def get_InstantCurrent(self, axis):
+        if axis == 1:
+            raise RuntimeError('The axis 1 does not use the extra attributes')
+        axis -= 1
+        cmd = 'CHAN{0:02d}:INSCurrent?'.format(axis)
+        return eval(self.sendCmd(cmd))
+
+
+###############################################################################
+#                Controller Extra Attribute Methods
+###############################################################################
+
+    @debug_it
+    @handle_error(msg="get_AcquisitionMode:")
+    def get_AcquisitionMode(self):
+        value = self.sendCmd('ACQU:MODE?')
+        return value
+
+    @debug_it
+    @handle_error(msg="set_AcquisitionMode:")
+    def set_AcquisitionMode(self, value):
+        self.sendCmd('ACQU:MODE %s' % value)
+
+    @debug_it
+    @handle_error(msg="get_PointsPerStep:")
+    def get_PointsPerStep(self):
+        return self._points_per_step
+
+    @debug_it
+    @handle_error(msg="set_PointsPerStep:")
+    def set_PointsPerStep(self, value):
+        self._points_per_step = value
```

Comment:
how one is supposed to update this value before an scan?

Reply:
from the dashboard or some view to configure!